aws/logs_monitoring/tools/list_layers.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Are you sure that the `list_layers.sh` and `add_new_region.sh` scripts are needed? It looks to me like the `publish_layers.sh` script uses all available regions.

Reply:
list_layers is useful for local development, even if it isn't strictly needed during the publish flow. add_new_region is useful because it grabs a current layer zip and redeploys that, without having to go back and rebuild from the last release commit.